core/src/main/java/dk/alexandra/fresco/framework/builder/numeric/field/BigIntegerModulus.java

```diff
@@ -29,7 +32,21 @@
   BigInteger getBigInteger() {
     return value;
   }
-
+  
+  /**
+   * Compute <i>x</i> modulus this modulus <i>M</i> assuming that <i>0 &le; x < M<sup>2</sup></i>
+   * 
+   * @param x A non-negative integer smaller than the modulus squared.
+   * @return A BigInteger equal to x mod M.
+   */
+  BigInteger reduceModThis(BigInteger x) {
+    // We don't serialize the reducer and create it lazily if needed
+    if (Objects.isNull(reducer)) {
```

Comment:
why not just == null?

Reply:
No reason. == null might be slightly more readable though, so I'll change it